src/app/routes/applications/new-role/new-role.component.ts

```diff
@@ -244,9 +243,9 @@ export class NewRoleComponent implements OnInit, AfterViewInit {
     return this.roleForm.get(control).hasError(errorType);
   }
 
-  alphaNumericOnly(event: any, includeDot?: boolean) {
+  alphaNumericOnly() {
     this.isExistsRoleName = false;
-    return isAlphaNumericOnly(event, includeDot);
+    return true;
```

Comment:
why do you need return true here? Btw it would be nice to add return type to functos

Reply:
If it would return false, then it would prevent from setting any value into input. Anyway i removed return value.

Btw. this is a workaround for role creation. This part is refactored [here](https://github.com/energywebfoundation/switchboard-dapp/pull/504), but i wanted to have it working right now as expected.

Reply:
Got it